proto/internal/temporal/server/api/matchingservice/v1/request_response.proto

```diff
@@ -641,3 +641,11 @@ message DescribeWorkerRequest {
 message DescribeWorkerResponse {
     temporal.api.worker.v1.WorkerInfo worker_info = 1;
 }
+
+message EnablePriorityAndFairnessRequest {
```

Comment:
We have so many rpcs.. I'm wondering if it would easier to make this another field of UpdateTaskQueueConfigRequest, like there could either be an updated_taskqueue_config, or a internal_set_fairness_state. I guess overall it's not that much simpler?

Anyway, can we name this SetFairnessState since that's a more direct description of what it does. And it should take a FairnessState to set, so it can be used for setting to V1 or V0.